### DISCUSSION TITLE
Duplication of upscalers when using symbolic links

### DISCUSSION THREAD

**Discussion by ScreNN** [lllyasviel/stable-diffusion-webui-forge#560] [Q&A]
I'm trying to organize all the models outside of the root SD folder, but in a separate one, for ease of use. But I've encountered a problem that it is the upscalers that start to be duplicated when using symbolic links
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/13724242/9df3cd3c-c54a-4ebb-a080-d8428338617b)
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/13724242/87cf726c-9a76-4842-ab72-f3c44a21a9c6)
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/13724242/06e8ee32-73d1-4263-9093-e9e77902a70e)

**Comment by catboxanon** [comment]
It will be fixed when the latest upstream is merged into this repo. See https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/313 for reference.
